### DISCUSSION TITLE
有没有人看得懂这张图出自什么游戏/冻鳗

### DISCUSSION THREAD

**Discussion by sjk624** [share121/inter-knot#2640] [其他]
![微信图片_20240811103234(1)](https://github.com/user-attachments/assets/850b0b7c-a07b-439f-933b-6dc2d5f797d9)

**Comment by Fintbo** [comment]
Muse Dash(即答)

**Reply by qmtqyh** [reply]
不是东方吗？这张图跟MD没关系吧

**Comment by sjk624** [comment]
刚发就被识破了，车万人无处不在<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by lixiao189** [comment]
钓符：这是什么动漫

**Reply by sjk624** [reply]
咬符：东方是一款由官方游戏，漫画组成的作品

**Comment by Sekiranu9** [comment]
是赤色杀人魔和日本高中生<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
